libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/drillDown.spec.ts

```diff
@@ -136,4 +159,88 @@ describe("Drilling", { tags: ["post-merge_integrated_bear"] }, () => {
             drillModal.getTable().getColumnValues(0).should("deep.equal", ["2011"]);
         });
     });
+
+    describe("Basic drill down",{ tags: ["checklist_integrated_bear"] }, () => {
+        const tableData = [
```

Comment:
Move the definition `const tableData = ....` to the top of the test class. Like:
```
const PRODUCT_ID = "1057";
const YEAR_CLOSE ="503";
const QUARTER_CLOSE ="522"
```